source/NeutronaWand/NeutronaWand.ino

```diff
@@ -782,14 +811,20 @@ void startVentSequence() {
     modeFireStop();
   }
 
-  // Turn on hat light 2.
-  digitalWrite(led_hat_2, HIGH);
+  // Turn on hat light 2 in 1984/1989 as overheat indicator; turn off in Afterlife/Frozen Empire.
+  if(getNeutronaWandYearMode() == SYSTEM_1984 || getNeutronaWandYearMode() == SYSTEM_1989) {
+    digitalWrite(led_hat_2, HIGH);
+  }
+  else {
+    digitalWrite(led_hat_1, LOW);
+    digitalWrite(led_hat_2, LOW);
+  }
 
-  delay(100); // Really should avoid this if possible since it will corrupt our ezButton states
+  delay(100); // TODO: Really should avoid this if possible since it will corrupt our ezButton states
```

Comment:
I haven't tried with it removed, and you're right, I'm not sure why it's there either. I didn't put it into Quick Vent and that seems to work fine, so I'll just remove it.

Speaking of not knowing why things are there, why do we delay 50ms before playing firing sounds and 100ms before playing end-firing sounds? It doesn't affect much except that if you tap Intensify and release it faster than 50ms, no firing sound will play but the end-firing sound will play. Not a major thing, but just wondered why it's there.

Reply:
I think that old delay was for some sound timing and I was trying to get things out quickly.

The ms timer delay for firing sounds is for helping the serial communication buffer to the audio board from not overflowing (or it was the original intention). If I recall some sounds play instantly while a few more get mixed in slightly after. There was a issue with the wav trigger track reporting as it always sends data back which was causing a major crash bug in September 2023. It was later fixed by fixing some serial data checks from incoming data. It shouldn’t be a issue for GPStar audio. Though it may be ok to remove those delays or make them super short, will need to be tested if it is decided to remove them though.

Reply:
Sounds like something to be left for after 5.0.0, then. Let's not break too much before a major release, hahaha. Thanks!

Looking into the cyclotron thing now; no, it's not intended behavior. I didn't think I'd touched the cyclotron except to call the function that reverts it to normal speed, so I'm not sure why it shuts off. Poking around.